firmware/integration/rusefi_config.txt

```diff
@@ -1060,7 +1060,7 @@ bit tempBooleanForVerySpecialLogic
 	float turboSpeedSensorMultiplier;;"mult", 1, 0, 0, 7000, 3
 
 	Gpio[CAM_INPUTS_COUNT iterate] camInputsDebug;
-	int16_t acIdleRpmBump;Extra idle target speed when A/C is enabled. Some cars need the extra speed to keep the AC efficient while idling.;"RPM", 1, 0, 0, 1000, 0
+	int16_t acIdleRpmBump;Idle target speed when A/C is enabled. Some cars need the extra speed to keep the AC efficient while idling.;"RPM", 1, 0, 0, 2000, 0
```

Comment:
"bump" in the method name is no longer appropriate

while renaming variable hopefully we would find not-zero default value which would need to be updated

Reply:
Hi, I have changed the names, default value and CHANGELOG, just not sure how to add this to this pull request 😅
https://github.com/pchmura4/rusefi/commit/5a948f2ae6bfcdb6730518ff937599c21118e1ea